lnrpc/gen_protos.sh

```diff
@@ -30,8 +30,7 @@ do
     DIRECTORY=$(dirname ${file})
     echo "Generating protos from ${file}, into ${DIRECTORY}"
 
-    protoc -I/usr/local/include -I. \
-           -I$GOPATH/src \
-           --go_out=plugins=grpc:. \
+    protoc -I/usr/local/include -I.\
+           --go_out=plugins=grpc,paths=source_relative:. \
```

Comment:
Should also update the other uses of `protoc` with these changes.

Reply:
I am not sure, because it still needs to find that external dep somewhere. `protoc` doesn't care about go modules I think.

Reply:
The other references to `GOPATH` are dependencies of `protoc`, not `lnd`. Those need to be installed somewhere. I think it is okay that those are installed in `GOPATH` as is also instructed in the readme.

Reply:
I think it worked for you because you also had `lnd` still in `GOPATH`? Only if you fully `rm -rf` lnd sources from there, you can see the difference.